auraed/src/lib.rs

```diff
@@ -37,6 +38,58 @@
 //! The Aurae Daemon (auraed) is the main server implementation of the Aurae
 //! Standard Library.
 //!
+//! The Aurae Daemon runs as a gRPC server which listens over a unix domain socket by default.
+//!
+//! ```bash
+//! /var/run/aurae/aurae.sock
+//! ```
+//!
+//! ## Running Auraed
+//!
+//! Running as `/init` is currently under active development.
+//!
+//! To run auraed as a standard library server you can run the daemon alongside your current init system.
+//!
+//! ```bash
+//! sudo -E auraed
+//! ```
+//!
+//! Additional flags are listed below.
+//!
+//! ```bash
```

Comment:
We should remove the bash example help as these are documented in other places automatically. We aren't gaining anything by adding it here except creating yet another place we have to manually keep in sync. 

Please remove ALL of the bash help output.